drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/window/DimensionsPatternTest.java

```diff
@@ -24,15 +24,15 @@
 
     @Test
     public void validSample() {
-        Matcher m = WindowResizer.DIMENSIONS_PATTERN.matcher("100x200");
+        Matcher m = Dimensions.DIMENSIONS_PATTERN.matcher("100x200");
```

Comment:
in this test, you can use the object `Dimensions` to verify no only the pattern, but also the rest of the class logic